src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/config.jelly

```diff
@@ -17,14 +17,15 @@
 <f:block>
```

Comment:
This file only appears to have whitespace changes. Can you remove it from the change list?